pom.xml

```diff
@@ -52,6 +56,19 @@
 			<artifactId>guava</artifactId>
 			<version>28.2-android</version>
 		</dependency>
+		<dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
+			<version>8.0.30</version>
+			<scope>compile</scope>
```

Comment:
Why scope is compile?

Reply:
I added these dependencies with the IDE, compile it is the default scope. Should I change it to another scope?

Reply:
Let's not to use scopes

Reply:
Ok, I will eliminate those